### PR TITLE
Releasing 0.30.0

### PR DESCRIPTION
**Pull request by JThramer**
image

/cc @mapbox/navigation-ios

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,11 +54,11 @@ Alternatively, to install Mapbox Navigation using [Carthage](https://github.com/
 
 1. Create a [Cartfile](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#github-repositories) with the following dependency:
    ```cartfile
-   github "mapbox/mapbox-navigation-ios" ~> 0.29
+   github "mapbox/mapbox-navigation-ios" ~> 0.30
    ```
    or with the following dependency for the latest prerelease:
    ```cartfile
-   github "mapbox/mapbox-navigation-ios" "v0.30.0-beta.1"
+   github "mapbox/mapbox-navigation-ios" "v0.30.0"
```
Comment (1ec5) [inline README.md]: I guess we can safely remove the two code samples about prereleases now.

File: README.md [hunk 2]
```diff
@@ -54,11 +54,11 @@ Alternatively, to install Mapbox Navigation using [Carthage](https://github.com/
 
 1. Create a [Cartfile](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#github-repositories) with the following dependency:
    ```cartfile
-   github "mapbox/mapbox-navigation-ios" ~> 0.29
+   github "mapbox/mapbox-navigation-ios" ~> 0.30
```
Comment (1ec5) [inline README.md]: custom-navigation.md also has similar code that needs updating.

Comment (1ec5) [reply README.md]: Never mind, it’s already been updated for v0.30.0.

Comment (JThramer) [reply README.md]: 😎

**Pull request merged**